test/integration/targets/get_url/tasks/main.yml

```diff
@@ -368,6 +368,13 @@
   async: 90
   poll: 0
 
+- name: Wait for SimpleHTTPServer to come up online
+  wait_for:
+    host: 'localhost'
+    port: '{{ http_port }}'
+    state: started
+  delegate_to: localhost
```

Comment:
We're trying to verify connectivity from the host under test to itself, so this is not necessary. Once we support split controller/remote testing it will be incorrect.

```suggestion
```

Reply:
Done.